### ISSUE TITLE
Design: People of WordPress _might_ be nicer to ensure we show headshots

### ISSUE THREAD

**Issue by jeffpaul** [WordPress/wporg-news-2021#254]
I'm not sure what the current logic is for the `People of WordPress` section but it _might_ be nicer to show all headshots and _if possible_ their name as an overlay on the image / caption below the image?  Right now the image for Devin Maeztri is kinda suboptimal as it crops out their actual photo.

![Screen Shot 2022-01-28 at 12 02 53 PM](https://user-images.githubusercontent.com/2818133/151598588-3d2efcd5-464c-4cac-9a35-a67ae31a18b0.png)

**Comment by coreymckrill** [comment]
Looks like a classic thumbnail cropping-to-the-center issue. We might be able to get a better thumbnail for Devin's in particular. The tricky part with adding just their name as an overlay is that we'd probably need to save that as a separate postmeta value. Alternatively, we could overlay the post title, which includes their name, but then it might cover up a significant portion of the image...

**Comment by coreymckrill** [comment]
@beafialho thoughts on this?

**Comment by iandunn** [comment]
There's some earlier discussion in #111

**Comment by beafialho** [comment]
I agree with @jeffpaul about showing people's faces, that was the intent of this section. Maybe that's something we can edit directly in the post content by adding headshots or using the post's featured image?

As for showing the names, thanks for pointing that out, I missed opening up an issue for that. They should show up on hover, similarly to what happens in the [Community](https://wordpress.org/news-test/category/community/) section.
